C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -0,0 +1,26 @@
+#include <bits/stdc++.h>
+using namespace std;
+int F(int n){
+    int arr[n+1],i,j; 
+    for(i=0;i<=n;i++){
+        arr[i]=i;
+    }
+    for(i=2;i<=n;i++){
+        if(arr[i]==i){             // checking if arr[i] has been untouched or not
+            for(j=i;j<=n;j+=i){
+                arr[j]-=arr[j]/i;  // here we are removing no. of coprime of j integer
+            }
+        }
+    }
+    // till now arr[i] shows no. of coprimes of n from 1 to n
+    for(i=2;i<=n;i++){
+        arr[i]+=arr[i-1]; // applying dp to calculate no. of coprime 
+    }                     // between 1 and n (both 1 and n inclusive)
+    return arr[n];
+}
+int main(){
+    int n;
+    cin>>n;
+    cout<<F(n)<<endl; // always remember 0 is only coprime with 1
+    return 0;        
+}
```

Comment:
Add sample Input/Output.

Reply:
Sir, I have added two sample cases at the end of code plus giving justification of dynamic programming